Cargo.toml

```diff
@@ -69,9 +69,9 @@ orchard = { version = "0.10.1", default-features = false }
 pasta_curves = "0.5"
 
 # - Transparent
-bip32 = { version = "0.5", default-features = false }
+bip32 = { version = "=0.6.0-pre.1", default-features = false }
 ripemd = { version = "0.1", default-features = false }
-secp256k1 = { version = "0.27", default-features = false, features = ["alloc"] }
+secp256k1 = { version = "0.29", default-features = false, features = ["alloc"] }
```

Comment:
Note to self: `secp256k1` and `bip32` need to be mentioned in the changelogs. I'll do that in a follow-up PR.

Reply:
https://github.com/zcash/librustzcash/pull/1689